### DISCUSSION TITLE
Suggestion for a Redesign of the Files Application

### DISCUSSION THREAD

**Discussion by myhousece** [elementary/files#2450] [Ideas]
### Problem

Suggestion for a Redesign of the Files Application
![arquivo_atual](https://github.com/user-attachments/assets/46aeb61f-a004-4efc-be79-40983f76a7e4)

### Proposal

Hello, elementary OS team,

First of all, I would like to congratulate you all on the excellent work you have done with elementary OS. In my opinion, it is one of the most beautiful and well-designed Linux distributions out there, and your work is truly admirable.

I am sending this suggestion with the intention of contributing to making the system even more cohesive and refined in its design. Recently, I created a redesign for the Files application, and I believe it can help improve the user experience and the aesthetics of the system. This redesign is based on the changes your team has been making within the settings area, aiming to make the overall system design even more cohesive and harmonious.

One of the changes I made was to the folder icon. I kept the same design but made a subtle change: I rounded the corners to align with the style of the "blue" folder icon in the applications area. This way, the icon remains in the same style but with a more modern touch that integrates better with the overall system look.

I am aware that a redesign is not a simple task and involves a lot of work, and my intention is simply to offer a small contribution to the excellent work you are already doing.

I hope you consider this suggestion and feel free to adjust it as you see fit.

Thank you in advance for your attention and for the incredible work you continue to do. I am excited to see how elementary OS will continue to evolve!

Best regards,
Reginaldo

![pasta elementary new_ok](https://github.com/user-attachments/assets/24da601d-4d43-4bdc-8a7e-b9ea0cf220c2)
![folder elementaryOS_8 new marcacoes](https://github.com/user-attachments/assets/60fe58ce-eaba-411d-b595-0b1ac7cc5602)

**Comment by jeremypw** [comment]
You have already raised an issue in the elementary/icons project regarding the folder icon for the design team to consider.
The other main suggestion is to split the header which would be consistent recent changes to other elementaryos apps: Settings, Code, Music, Calendar for example. But not Photos (atm).  Of these Code and Photos can hide the left sidebar.  Files used to be able to hide the sidebar (and there is still a setting for that) but this no longer happens - the code has been removed.  Clearly, if the sidebar may be hidden then the sidebar header cannot contain any essential controls.

I do not see any reason not to split the header in the case of Files, if the design team agree.  It may change the minimum sidebar width, depending on the controls in the header.  At the moment the minimum width is set by the width of the "Connect to Server" button.

For comparison, Gnome Files has a split header and a fixed width sidebar.

**Comment by jeremypw** [comment]
See https://github.com/elementary/icons/issues/1305 for associated issue about the icons.
See https://github.com/elementary/files/issues/2456 for associated issue about the sidebar
